docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc

```diff
@@ -0,0 +1,221 @@
+= Reinitialize a storage disk
+
+[abstract]
+--
+Steps to reinitialize an existing, but corrupted storage disk of an OpenShift 4 cluster on https://www.exoscale.com[Exoscale].
+--
+
+== Starting situation
+
+* You already have a OpenShift 4 cluster on Exoscale
+* You have admin-level access to the cluster
+* You want to reinitialize a corrupted storage disk of an existing storage node of the cluster.
+
+== Prerequisites
+
+The following CLI utilities need to be available locally:
```

Comment:
What are the typical symptoms of a corrupted disk? How would I verify that I have such a case and should follow this guide?

Reply:
added a tip box describing the main symptoms indicating a corrupted storage disk